edge/pkg/devicetwin/dtmanager/communicate.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtcommon"
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dtcontext"
 	"github.com/kubeedge/kubeedge/edge/pkg/devicetwin/dttype"
+	uuid "github.com/satori/go.uuid"
```

Comment:
It would be good change, while i am concern is which `uuid` lib should we use:
```
$ grep uuid go.mod
	github.com/google/uuid v1.1.1
	github.com/pborman/uuid v1.2.0
	github.com/satori/go.uuid v1.2.0

```

Reply:
The import uuid is refer other packages existing in the project. 
If change to the following: github.com/satori/go.uuid/v1.2.0, the IDE will require to renew sync dependencies of github/kubeedge/kubeedge.

Reply:
Track as #2330

Reply:
We can use `github.com/google/uuid` consistent with Kubernetes upstream and can switch all uuid ref to   `github.com/google/uuid` in follow-up PR.

Reply:
OK, related content has been modified.